atlascope/core/job_types/__init__.py

```diff
@@ -0,0 +1,9 @@
+from . import average_color, brightest_n_pixels
+
+available_job_types = {
+    module.__name__.replace(module.__package__, '')[1:]: module.run
```

Comment:
Is that `[1:]` meant to strip off a dot? If so, then I think this might be clearer:

```suggestion
    module.__name__.replace(f"{module.__package__}.", ''): module.run
```

but this feels pretty creaky too so let's just keep an eye on it.

A more solid but less automated way to handle this would be to manually list name/function pairs in this declaration. Perhaps fewer surprises to do it that way.